src/HasWishlists.php

```diff
@@ -4,9 +4,27 @@
 
 use DB;
 use Exception;
+use Illuminate\Support\Facades\Auth;
 
 trait HasWishlists
 {
+    /**
+     * Count wishlist for user.
+     *
+     * @param  array  $collections
+     * @return int
+     */
+    public function countWishlist(array $collections = []): int
```

Comment:
Can you make this a little more flexibel by making it possible the either add an array (for multiple) or a string (for a single) collection.